src/server/wkPlaywright.ts

```diff
@@ -283,7 +283,10 @@ function wrapTransportWithWebSocket(transport: ConnectionTransport) {
     }
     socket = s;
     s.on('message', message => transport.send(Buffer.from(message).toString()));
-    transport.onmessage = message => s.send(message);
+    transport.onmessage = message => {
+      if (s.readyState === ws.OPEN)
```

Comment:
It'd be a little more accurate if we checked that it's ws.CLOSING and send the message otherwise?

Reply:
Done.